src/components/Requirements.vue

```diff
@@ -1,149 +1,163 @@
+          <div v-bind:class="{'active': minor.display}"  @click="activate(id)" class="major-title" v-for="(minor, id) in minors" :key="minor.id">
+            <p v-bind:class="{'active': minor.display}"  class="minor-title-top">{{minor.minorFN}}</p>
+            <p v-bind:class="{'active': minor.display}" class="minor-title-bottom">{{user.collegeFN}}</p> <!-- Change for multiple colleges -->
+          </div>
         </div>
-      </div>
 
-      <!--Show more of completed requirements -->
-      <div v-if="req.displayDetails">
-        <p class="sub-title">On-Going Requirements</p>
-        <div class="separator"></div>
-        <div
-          v-for="(subReq, id) in req.ongoing"
-          :key="subReq.id">
-          <div class="row depth-req">
-            <div class="col-1" @click="toggleDescription(index, 'ongoing', id)">
-              <button class="btn">
+        <!-- progress bar settings -->
+        <div v-if="showMajorOrMinorRequirements(index, req.group)" >
+          <div class="progress">
+            <div
+              class="progress-bar"
+              :style="{ 'background-color': `#${req.color}`, width: `${(req.fulfilled/req.required)*100}%`}"
+              role="progressbar"
+            ></div>
+          </div>
+
+          <p class="progress-text">
+            <strong>{{ req.fulfilled }}/{{ req.required }}</strong>
+            Total {{ req.type }} Inputted on Schedule
+          </p>
+
+          <!--View more college requirements -->
+          <div class="row top">
+            <div class="col-1 p-0" >
+              <button :style="{ 'color': `#${req.color}` }" class="btn" @click="toggleDetails(index)" style="color:#1AA9A5;">
                 <!-- svg for dropdown icon -->
                 <img
-                  v-if="subReq.displayDescription"
+                  v-if="req.displayDetails"
                   class="arrow arrow-down"
-                  src="@/assets/images/dropdown.svg"
+                  src="@/assets/images/dropdown-blue.svg"
                   alt="dropdown"
                 />
                 <img
                   v-else
                   class="arrow"
-                  src="@/assets/images/dropright.svg"
+                  src="@/assets/images/dropright-blue.svg"
                   alt="dropdown"
                 />
               </button>
             </div>
-            <div class="col-7" @click="toggleDescription(index, 'ongoing', id)">
-              <p class="sup-req pointer">{{subReq.requirement.name}}</p>
+            <div class="col p-0 ">
+              <button class="btn req-name" :style="{ 'color': `#${req.color}` }" @click="toggleDetails(index)">{{ (req.displayDetails) ? "HIDE" : "VIEW" }} ALL {{ req.group }} REQUIREMENTS</button>
             </div>
-            <div class="col">
-              <p class="sup-req-progress text-right">
-                ( {{
-                  (subReq.requirement.fulfilledBy !== 'self-check')
```

Comment:
Was this and a bunch of other things in this file meant to be changed? This deletion seems to be causing every requirement to be a "self-check," and if I switch the code back to this from what it was changed to, it fixes it. Do you know if that is the only change that needs to be reverted, or is a lot of the code in here overwriting things that should not be?

Reply:
Ahh yeah I had deleted this before I realized the Reqs Json had changed quite a bit-- should be fixed now.